client/js/components/day/day.js

```diff
@@ -18,9 +18,8 @@ var Day = React.createClass({
 
   componentWillMount: function(){
     DayStore.addChangeListener(this._handleUpdate);
-     // trigger API call to get initial data. Date.now will
-     // change once we get routing sorted
-    ViewActionCreator.getDayData(Date.now());
+     // trigger API call to get initial data.
+    ViewActionCreator.getDayData();
```

Comment:
yep! I am going to make a PR today which includes day-header, adding the functionality to switch between days - the future pull request includes passing different dates using moment.js